src/main/kotlin/org/rust/lang/core/resolve/ref/RsPathReferenceImpl.kt

```diff
@@ -81,6 +85,14 @@ fun resolvePath(path: RsPath, lookup: ImplLookup = ImplLookup.relativeTo(path)):
         }
     }
 
+    val lifetimeArguments: List<Re>? =
```

Comment:
You can remove `?` from type if you're returning `orEmpty()`, I guess

Reply:
For consistency with `typeArguments` I have removed `orEmpty()`.